src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx

```diff
@@ -6,13 +6,15 @@ const DictionaryDetailCard = (dictionary) => {
   const {
     dictionary: {
       name, created_on, updated_on, public_access, owner, owner_type, active_concepts, description,
-      owner_url, short_code, default_locale,
+      owner_url, short_code, default_locale, url,
     },
   } = dictionary;
   const DATE_OPTIONS = {
     weekday: 'long', year: 'numeric', month: 'short', day: 'numeric', hour: 'numeric', minute: 'numeric',
   };
 
+  const traditionalUrl = `https://qa.openconceptlab.org${url}`;
+
```

Comment:
The whitespaces here are not consistent with rest of the file.

Reply:
Noted. Thank you for that observation.